src/components/Card/index.tsx

```diff
@@ -0,0 +1,126 @@
+import { FC } from 'react';
+import classNames from '@/components/Card/card.module.scss';
+
+type Props = {
+  title: {
+    text: string,
+    link?: string
+  },
+  data: {
+    key: string,
+    value: string
+  }[],
+  owner?:
+  {
+    userName: string,
+    imgUrl: string,
+  },
+  participants?:
+  {
+    firstName: string,
+    lastName: string,
+    userName: string,
+    imgUrl: string,
+  }[],
+  button?: {
+    text: string,
+    link?: string,
+    onClick?: () => void
+  };
+};
+
+const Card: FC<Props> = ({
+  title, data, owner = undefined, participants = undefined, button = undefined,
+}) => {
+  const { text: tileText } = title;
+
+  function informationElement(key: string, value: string) {
+    return (
+      <span className={classNames.statusElement}>
+        <span className={classNames.statusLable}>{`${key}: `}</span>
+        <strong>{value}</strong>
+      </span>
+    );
+  }
```

Comment:
@ASHWINIAJIT Please use the arrow function.

Reply:
Done!

https://github.com/Real-Dev-Squad/website-status/pull/101/commits/831d76531845941f32099c732993abf48ad1b80e